models/model_runner.py

```diff
@@ -7,7 +7,7 @@
 
 from DataProcessing.DB.dal import Crop, get_entries
 from DataProcessing.utils import viz_DB_data_on_video
-from models.model_constants import *
```

Comment:
I had the same issue on Master --- it could be solved by marking collect root paths on pycharm
![image](https://user-images.githubusercontent.com/31073837/195805171-abe80a6f-8407-4d8d-84af-ba66b7c482a6.png)